app/views/site/id.html.erb

```diff
@@ -15,7 +15,15 @@
      data[:token] = token.token
    end
    data[:locale] = ID::LOCALES.preferred(preferred_languages).to_s
-   data[:asset_map] = assets("iD").to_json %>
+   data[:asset_map] = assets("iD").to_json
+
+  # Include user preferences for iD editor
+   if current_user
+     user_preferences = current_user.preferences.each_with_object({}) do |pref, hash|
```

Comment:
Do we really need all user properties for iD? This could be a fair amount of data. How about only adding properties only that are relevant for iD?